rakefile.rb

```diff
@@ -12,73 +11,10 @@
 
 task :prepare_for_build, [:build_version] do |t, args|
   update_package_version(args.build_version, description_file)
-  copy_files_to_lib_folder
   install_pksim('develop')
 end
 
-task :postclean do 
-  os = 'win'
-  clear_folders
-  nuget_restore os
-  copy_files_to_lib_folder 
-end
-
-# This task is temporary until we have an automated linux build
-task :create_linux_build, [:product_version, :build_dir, :linux_distro] do |t, args|
-  product_version = sanitized_version(args.product_version)
-
-  build_dir = args.build_dir
-  linux_distro = args.linux_distro
-
-  #run nuget to get linux packages
-  nuget_restore linux_distro
-
-  tar_file_name = "ospsuite_#{product_version}.tar.gz"
-  
-  # Tar file produced by the script
-  tar_file = File.join(build_dir, tar_file_name)
-
-  #unzip it in a temp folder
-  temp_distro_dir = File.join(temp_dir, linux_distro)
-  FileUtils.mkdir_p temp_distro_dir
-
-  command_line = %W[xzf #{tar_file} -C #{temp_distro_dir}]
-  Utils.run_cmd('tar', command_line)
-
-  ospsuite_dir = File.join(temp_distro_dir,  'ospsuite')
-  inst_lib_dir = File.join(ospsuite_dir, 'inst', 'lib')
-
-  #Remove the windows dll that should be replace by linux binaries
-  delete_dll('OSPSuite.FuncParserNative', inst_lib_dir)
-  delete_dll('OSPSuite.SimModelNative', inst_lib_dir)
-  delete_dll('OSPSuite.SimModelSolver_CVODES', inst_lib_dir)
-
-  #Copy the linux binaries
-  copy_so('OSPSuite.FuncParser', linux_distro, inst_lib_dir)
-  copy_so('OSPSuite.SimModel', linux_distro,  inst_lib_dir)
-  copy_so('OSPSuite.SimModelSolver_CVODES', linux_distro,  inst_lib_dir)
-
-  Dir.chdir(temp_distro_dir) do
-    tar_archive_name = "ospsuite_#{product_version}_#{linux_distro}.tar"
-    command_line = %W[cf #{tar_archive_name} ospsuite]
-    Utils.run_cmd('tar', command_line)
-
-    Utils.run_cmd('gzip', %W[#{tar_archive_name}])
-
-    FileUtils.mv("#{tar_archive_name}.gz", build_dir)
-  end
-end
-
-
 private
-def copy_so(file, linux_distro, target_dir)
-  native_folder = '/bin/native/x64/Release/'
-  copy_dependencies packages_dir, target_dir do
-    copy_files "#{file}.#{linux_distro}*/**/#{native_folder}", 'so'
-    copy_files "#{file}.#{linux_distro}*/**/netstandard*", 'dll'
-  end
-end
-
 def install_pksim(branch)
   file_name ='setup.zip'
```

Comment:
I think PK-Sim is required for some tests that use creation of individuals. Maybe we should consider at some point to completely remove the dependency by separating the "create..." algorithms in separate submodule?